docs/did-method-spec.html

```diff
@@ -194,7 +197,7 @@ <h4 id="fees">Fees and Rent</h4>
     </li>
   </ul>
   <p>By default, the fee payer is the owner of the DID that will be created. However, the implementation supports
-  a separate fee payer, that will incur the charges of creation/updating/deletion of the DID without having
+  a separate fee payer, that will incur the charges of registration/updating/deletion of the DID without having
   any permissions on the DID itself. This allows a central body to generate DIDs for its customers, without
   requiring them to have a SOL balance.</p>
   <h3>Read (Resolve)</h3>
```

Comment:
"<p>A SOL DID record can be looked up on Solana,by stripping the did:sol: prefix from the DID, and
    network identifier if present, and looking up the account on-chain.</p>"

I would rephrase this to:

"<p>A SOL DID record can be looked up on Solana,by stripping the did:sol: prefix from the DID, and
    network identifier if present, and looking up the resultant account on-chain. If no data is present on-chain, the generated DID document (see above) should be returned.</p>

Reply:
I would also suggest adding details about the default information. Specifically something like:

The authority is a special key from which the DID is derived. It is always present in the verification methods list, and by default is added as the only key in the capabilityInvocations list, indicating that it is permitted to change the DID Document contents. It can, however be removed from the list by submitting an update instruction where the capabilityInvocation list is explicitly set (e.g. to the empty list or some other list of keys)

Also add links to the above where appropriate e.g. around capabilityInvocation pointing to the w3c did document spec and around update instruction pointing to the update section.